### PR TITLE
fix: updated universal links

### PR DESCRIPTION
**Pull request by nlewis84**
Fixed an issue with Universal Links and confirmed via testing below.

Screen Shot 2021-09-15 at 1 24 49 PM
Screen Shot 2021-09-15 at 1 25 29 PM

https://user-images.githubusercontent.com/72768221/133488953-e1546809-3022-4e42-9138-9abdb089ffed.mp4

### CODE REVIEW

File: apolloschurchapp/android/app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -36,7 +36,7 @@
 
           <!-- Handle urls starting with "http://www.example.com/products" -->
           <data android:scheme="https"
-                android:host="@string/APP_DATA_URL"
+                android:host="rivervalley-herokuapp-com.global.ssl.fastly.net"
```
Comment (redreceipt) [inline apolloschurchapp/android/app/src/main/AndroidManifest.xml:39]: these should be heroku links right?

Comment (nlewis84) [reply apolloschurchapp/android/app/src/main/AndroidManifest.xml:39]: I actually tested this fastly one several times (which was already in place) and didn't notice any issues with it. Can definitely change all these references to the heroku instance.

Comment (nlewis84) [reply apolloschurchapp/android/app/src/main/AndroidManifest.xml]: I went ahead and changed it!  Tested as well and all looks good. I'm gonna go ahead and push this to master. I feel good about these changes.

**Comment by nlewis84** [comment]
@conrad-vanl or @vinnyjth I want to git this merged before I merge Michael's other PR. Can one y'all check this out?

**Review by vinnyjth — APPROVED** [review]

**Pull request merged**